mod/notifications/start.php

```diff
@@ -113,6 +114,63 @@ function _notifications_page_menu($hook, $type, $return, $params) {
 }
 
 /**
+ * Register menu items for the page menu on group profiles
+ *
+ * @param \Elgg\Hook $hook hook
+ *
+ * @return array
+ *
+ * @access private
+ *
+ * @since 3.0
+ */
+function _notifications_groups_subscription_page_menu(\Elgg\Hook $hook) {
```

Comment:
Do we really need these expensive operations? I always find this menu confusing, would be easier to just say Manage group notifications, and be done with it.